src/monsters.cpp

```diff
@@ -37,6 +36,13 @@ extern Spells* g_spells;
 extern Monsters g_monsters;
 extern ConfigManager g_config;
 
+spellBlock_t::~spellBlock_t()
+{
+	if (spell && combatSpell) {
```

Comment:
You don't need to check if spell isn't null. It's safe to call delete on a null pointer.